compact/tracker.xml

```diff
@@ -28,7 +28,7 @@
     <comment> barrel length: 2R/tan(angle) - dz </comment>
 
     <comment> Position in center of space behind DIRC </comment>
-    <constant name="OuterMPGDBarrel_rmin"            value="EcalBarrel_rmin - BarrelExtraSpace_thickness / 2."/>
+    <constant name="OuterMPGDBarrel_rmin"            value="74.5*cm"/>
```

Comment:
We should still use the main parametrization, not hardcoded numbers. I'd say we should rename `BarrelExtraSpace_thickness` to `OuterMPGDBarrel_thickness` of 2 cm in `definitions.xml`. Essentially, what's in the detector geometry table should be defined in `definitions.xml`. @sly2j for comment (though he's busy today).

Reply:
Here: https://github.com/eic/epic/blob/barrel-mpgd-relocation/compact/definitions.xml#L468

Reply:
I agree with keeping this parameterized with OuterMPGDBarrel_thickness. Do we want to keep this layer parameterized via EcalBarrel_rmin or should this eventually be changed to be parametrized relative to the DIRC position. Although I guess if both DIRC and this outer MPGD are parameterized via the Ecal than there is no issue keeping EcalBarrel_rmin.

Reply:
I think we should not subtract anything from `EcalBarrel_rmin` (defined as [1]). Instead, let's define `PIDBarrel_rmin` = `CentralTrackingRegion_rmax` and go from there.

[1] https://github.com/eic/epic/blob/ec158412e9f3958ce7c17ee9f147c9479b84633f/compact/definitions.xml#L530

Reply:
I agree, we're parametrizing from inside out, so mixing in outside-in parametrization will make things a bit harder to work with down the road.